checks.d/ceph.py

```diff
@@ -82,20 +83,70 @@ def _extract_metrics(self, raw, tags):
             self.log.debug('Error retrieving osdperf metrics')
 
         try:
-            for osdinfo in raw['osd_pool_stats']:
-                name = osdinfo['pool_name']
-                local_tags = tags + ['ceph_pool:%s' % name]
+            # Health summary will be empty if no bad news
+            if raw['health_detail']['summary'] == []:
+                health = {'num_near_full_osds' : 0, 'num_full_osds' : 0}
+                self._publish(health, self.count, ['num_near_full_osds'], tags)
+                self._publish(health, self.count, ['num_full_osds'], tags)
+            else:
+                for osdhealth in raw['health_detail']['detail']:
+                    osd, pct = self._osd_pct_used(osdhealth)
+                    local_tags = tags + ['ceph_osd:%s' %  osd.replace('.','')]
```

Comment:
flake8 is not happy with the extra space afther the `%`: `./checks.d/ceph.py:94:57: E222 multiple spaces after operator`